packages/compass-e2e-tests/helpers/commands/create-index.ts

```diff
@@ -34,7 +34,22 @@ export async function createIndex(
   const indexName = extraOptions?.indexName ?? `${fieldName}_${indexType}`;
 
   // Open the modal
-  await browser.clickVisible(Selectors.CreateIndexButton);
+  const isWithCreateDropdown = await browser
+    .$(Selectors.CreateIndexDropdownButton)
+    .isExisting();
+  if (!isWithCreateDropdown) {
```

Comment:
This is very nitpicky, but as a style thing can we please `if (true) {} else {}` rather than `if (!true) {} else {}`. It find this way around so confusing 😆